src/core/style/_elevation.scss

```diff
@@ -0,0 +1,186 @@
+/**
+ * A collection of mixins and CSS classes that can be used to apply elevation to a material
+ * element.
+ * See: https://www.google.com/design/spec/what-is-material/elevation-shadows.html
+ * Examples:
+ *
+ * ```scss
+ * .md-foo {
+ *   @include $md-elevation(2);
+ *
+ *   &:active {
+ *     @include $md-elevation(8);
+ *   }
+ * }
+ * ```
+ *
+ * ```html
+ * <div id="external-card" class="md-elevation-z2"><p>Some content</p></div>
```

Comment:
If I understand your question correctly (which I _think_ I do but I'm not 100%), I imagine that if the user imports a vendor style such as those from `ngMaterial` once in a top-level bootstrap file or a `vendor.ts`-esque file, then these styles will only be loaded once and there won't be any `ViewEncapsulation` issues. 

I'm not as familiar with how angular caches loaded styles but would the framework be smart enough not to load styles more than once if the `ViewEncapsulation` is disabled? If not, let me know what makes the most sense in terms of exporting these styles.